numba/tests/support.py

```diff
@@ -19,6 +19,7 @@
 import ctypes
 import multiprocessing as mp
 import warnings
+import traceback
```

Comment:
unused import.

Reply:
It was a missing import that I added for
https://github.com/numba/numba/blob/0d8266613e568b344c13a8d2ee3937e9510d1b49/numba/tests/support.py#L841